CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
This can be lowered to 3.6 at least, couldn't it?

Reply:
Certainly :smile:. I just tested it with 3.0, and seems to be fine.